src/back/spv/mod.rs

```diff
@@ -1,3 +1,8 @@
+/*! Standard Portable Intermediate Representation (SPIR-V) backend
+ *
+ * Generates SPV-1.3 bytecode.
```

Comment:
This still stands? Can't we potentially generate SPV-1.5 now too? Just wondering and giving a thought.

Reply:
gonna fix this now!